src/commands/test/test_command_test.py

```diff
@@ -0,0 +1,59 @@
+# pylint: disable=invalid-name
+import re
+from dataclasses import fields
+from pathlib import Path
+from typing import cast
+from unittest.mock import MagicMock
+
+import pytest
+from pytest_mock import MockerFixture
+
+from src.commands.test import TestCommand
+
+
+@pytest.mark.asyncio
+async def test_test_command_runs_scheduler_properly(mocker: MockerFixture):
+    args = TestCommand.Args(
+        target=Path("foo"),
+        match=re.compile("foo"),
+        omit=re.compile("bar"),
+        cairo_path=[Path() / "baz"],
+    )
+    TestCollectorMock = mocker.patch(
+        "src.commands.test.test_command.TestCollector",
+    )
+    TestSchedulerMock = mocker.patch("src.commands.test.test_command.TestScheduler")
+    TestingLiveLogger = mocker.patch("src.commands.test.test_command.TestingLiveLogger")
+    project_mock = mocker.MagicMock()
+    protostar_directory_mock = mocker.MagicMock()
+    protostar_directory_mock.add_protostar_cairo_dir.return_value = args.cairo_path
+    test_command = TestCommand(project_mock, protostar_directory_mock)
+
+    await test_command.run(args)
+
+    cast(MagicMock, TestCollectorMock.return_value.collect).assert_called_once()
+    TestingLiveLogger.assert_called_once()
+    cast(MagicMock, TestSchedulerMock.return_value.run).assert_called_once()
+    assert (
+        str(args.cairo_path[0])
+        in cast(MagicMock, TestSchedulerMock.return_value.run).call_args_list[0][1][
+            "include_paths"
+        ]
+    )
+
+
+def test_test_command_name(mocker: MockerFixture):
+    test_command = TestCommand(mocker.MagicMock(), mocker.MagicMock())
+
+    assert test_command.name == "test"
+    assert len(test_command.description) > 3
+    assert test_command.example
```

Comment:
This test is useless, we are just checking hardcoded data, please remove it introduces unnecessary burden.

Reply:
Yep, I've added it to increase coverage.